web/packages/teleterm/build_resources/installer.nsh

```diff
@@ -13,6 +13,15 @@
     # Make EnVar define user env vars instead of system env vars.
     EnVar::SetHKCU
     EnVar::AddValue "Path" $INSTDIR\resources\bin
+
+    nsExec::ExecToStack '"$INSTDIR\resources\bin\tsh.exe" vnet-install-service'
+    Pop $0 # ExitCode
+    Pop $1 # Output
+    ${If} $0 != 0
+        MessageBox MB_ICONSTOP \

```

Comment:
Won't this just hang builds until they timeout since our Windows builds don't run in an interactive environment with someone available to dismiss the messagebox?

Reply:
this runs in the installer and displays the message to the user installing the connect, it doesn't run at build time